graylog2-web-interface/src/views/components/visualizations/worldmap/bindings.tsx

```diff
@@ -16,12 +16,68 @@
  */
 import type { VisualizationType } from 'views/types';
 
+import Viewport from 'views/logic/aggregationbuilder/visualizations/Viewport';
+import WorldMapVisualizationConfig from 'views/logic/aggregationbuilder/visualizations/WorldMapVisualizationConfig';
 import WorldMapVisualization from 'views/components/visualizations/worldmap/WorldMapVisualization';
 
+export type WorldMapVisualizationConfigFormValues = {
+  zoom: number,
+  centerX: number,
+  centerY: number,
+};
+
+const DEFAULT_FORM_VALUES = {
+  zoom: 1,
+  centerX: -0.3515602939922709,
```

Comment:
From what I can see, we can just use `0`/`0` for `centerX`/`centerY`, having the same result, but being slightly less magical.

Reply:
These are the default `react-leaflet` values, which we used in the past if the user did not changed the default viewport. But you are right, we can also just `0`/`0`, since the difference is not noticeable.